From: intrigeri <intrigeri@boum.org>
Date: Sun, 28 Jan 2018 18:51:40 +0000
Subject: Drop spurious trailing whitespace.

---
 apparmor/local/torbrowser.Browser.firefox | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/apparmor/local/torbrowser.Browser.firefox b/apparmor/local/torbrowser.Browser.firefox
index 2bbf71e..da8acb0 100644
--- a/apparmor/local/torbrowser.Browser.firefox
+++ b/apparmor/local/torbrowser.Browser.firefox
@@ -1,2 +1,2 @@
 # Site-specific additions and overrides for torbrowser.Browser.firefox.
-# For more details, please see /etc/apparmor.d/local/README. 
+# For more details, please see /etc/apparmor.d/local/README.
