From: intrigeri <intrigeri@boum.org>
Date: Thu, 26 Oct 2017 11:12:52 +0000
Subject: AppArmor: grant access to mostly innocuous stuff Firefox tries to read.
Forwarded: https://github.com/micahflee/torbrowser-launcher/pull/294
---
 apparmor/torbrowser.Browser.firefox | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/apparmor/torbrowser.Browser.firefox b/apparmor/torbrowser.Browser.firefox
index 39ac6a2..6a2c148 100644
--- a/apparmor/torbrowser.Browser.firefox
+++ b/apparmor/torbrowser.Browser.firefox
@@ -31,6 +31,10 @@
   deny /etc/machine-id r,
   deny /var/lib/dbus/machine-id r,
 
+  /dev/ r,
+  /dev/shm/ r,
+
+  owner @{PROC}/@{pid}/fd/ r,
   owner @{PROC}/@{pid}/mountinfo r,
   owner @{PROC}/@{pid}/stat r,
   owner @{PROC}/@{pid}/status r,
