From: intrigeri <intrigeri@boum.org>
Date: Mon, 29 Jan 2018 07:24:29 +0000
Subject: Fix comment.

---
 apparmor/torbrowser.Browser.plugin-container | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/apparmor/torbrowser.Browser.plugin-container b/apparmor/torbrowser.Browser.plugin-container
index f5554db..5fd3efa 100644
--- a/apparmor/torbrowser.Browser.plugin-container
+++ b/apparmor/torbrowser.Browser.plugin-container
@@ -4,7 +4,7 @@
 profile torbrowser_plugin_container {
   #include <abstractions/gnome>
 
-  # Uncomment the following lines if you don'want the Tor Browser
+  # Uncomment the following lines if you want Tor Browser
   # to have direct access to your sound hardware. You will also
   # need to remove the "deny" word in the machine-id lines further
   # bellow.
