From: Roger Shimizu <rosh@debian.org>
Date: Wed, 9 Sep 2020 01:46:15 +0900
Subject: Use /usr/bin/gpg instead of /usr/bin/gpg2

Closes: #969901
---
 torbrowser_launcher/common.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/torbrowser_launcher/common.py b/torbrowser_launcher/common.py
index 9cff5dc..f3cd13a 100644
--- a/torbrowser_launcher/common.py
+++ b/torbrowser_launcher/common.py
@@ -203,7 +203,7 @@ class Common(object):
         else:
             print('Refreshing local keyring...')
 
-        p = subprocess.Popen(['/usr/bin/gpg2', '--status-fd', '2',
+        p = subprocess.Popen(['/usr/bin/gpg', '--status-fd', '2',
                               '--homedir', self.paths['gnupg_homedir'],
                               '--keyserver', 'hkps://keys.openpgp.org',
                               '--refresh-keys'], stderr=subprocess.PIPE)
