From: Roger Shimizu <rosh@debian.org>
Date: Sun, 25 Apr 2021 22:51:12 +0900
Subject: Update apparmor profile

Closes: #980155
---
 apparmor/torbrowser.Browser.firefox | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/apparmor/torbrowser.Browser.firefox b/apparmor/torbrowser.Browser.firefox
index 57c0359..095b110 100644
--- a/apparmor/torbrowser.Browser.firefox
+++ b/apparmor/torbrowser.Browser.firefox
@@ -90,6 +90,7 @@ profile torbrowser_firefox @{torbrowser_firefox_executable} {
   /usr/share/gnome/applications/ r,
   /usr/share/gnome/applications/kde4/ r,
   /usr/share/poppler/cMap/ r,
+  /etc/xdg/mimeapps.list r,
 
   # Distribution homepage
   /usr/share/homepage/ r,
@@ -121,6 +122,7 @@ profile torbrowser_firefox @{torbrowser_firefox_executable} {
   deny @{HOME}/.cache/fontconfig/** rw,
   deny @{HOME}/.config/gtk-2.0/ rw,
   deny @{HOME}/.config/gtk-2.0/** rw,
+  deny @{PROC}/@{pid}/cgroup r,
   deny @{PROC}/@{pid}/net/route r,
   deny /sys/devices/system/cpu/cpufreq/policy[0-9]*/cpuinfo_max_freq r,
   deny /sys/devices/system/cpu/*/cache/index[0-9]*/size r,
